Remotion/Web/Core/Res/Themes/NovaViso/Image/sprite.svg

```diff
@@ -63,13 +70,27 @@
+  <svg class="sprite-part" id="DropDownMenuRequestInProgressIndicator" width="16" height="16" viewBox="0 0 16 16">
+    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+      <path d="M11.5,2.5 L11.5,4 C11.5,4.96649831 11.1082492,5.84149831 10.4748737,6.47487373 C9.84149831,7.10824916 8.96649831,7.5 8,7.5 C7.03350169,7.5 6.15850169,7.10824916 5.52512627,6.47487373 C4.89175084,5.84149831 4.5,4.96649831 4.5,4 L4.5,4 L4.5,2.5 L11.5,2.5 Z" stroke="#555555" fill="#FFFFFF"/>
```

Comment:
000000 instead of 555555 for 
ProgressIndicator Dropdown (Web) and Autocomplete (ObjectBinding)
Required marker

reasoning: the calender is also black.